katalogus/views/katalogus.py

```diff
@@ -37,22 +46,29 @@ def get_queryset(self):
         if "filter_options" in self.request.GET:
             filter_options = self.request.GET.get("filter_options")
             queryset = self.filter_queryset(queryset, filter_options)
+        if "sorting_options" in self.request.GET:
+            sorting_options = self.request.GET.get("sorting_options")
```

Comment:
Since this key is checked and for sure exists in the dictionary, you can simply access it like this:
```suggestion
            sorting_options = self.request.GET["sorting_options"]
```